diagnostic.md

```diff
@@ -6,65 +6,74 @@ Place your responses inside the fenced code-blocks where indicated by comments.
 What are two ways of telling rails where to route a GET request to '/books/4'?
 
 ```rb
-// your response here
+resources :books, only: [:show]
+books#show
+
+or did you want the curl request code?
 ```
 
 In what file does the above code go?
 
 ```md
-// your response here
+routes.rb
 ```
 
 In your own words, what does the command `bundle exec rake db:migrate` do?
 
 ```md
-// your response here
+It defines how the database should organize the hash data (model) into the database.
```

Comment:
It executes any code found within the Active Record migration files in db/migrate. The migrations make it so that we don't have to modify our database directly.